astropy_iers_data/__init__.py

```diff
@@ -1,4 +1,41 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-from .version import __version__
-__all__ = []
+import os
+
+from ._version import __version__
+
+__all__ = [
+    "IERS_A_FILE",
+    "IERS_A_URL",
+    "IERS_A_URL_MIRROR",
+    "IERS_A_README",
+    "IERS_B_FILE",
+    "IERS_B_URL",
+    "IERS_B_README",
+    "IERS_LEAP_SECOND_FILE",
+    "IERS_LEAP_SECOND_URL",
+    "IETF_LEAP_SECOND_URL"
+]
+
+DATA = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
```suggestion
DATA = pathlib.Path(__file__).resolve().parent / "data"
```

Pathlib has a lot of nice features over raw strings.